build.proj

```diff
@@ -515,142 +437,100 @@
   </Target>
 
   <Target Name="ChangeLogCheck">
-    <Exec Command="&quot;$(PowerShellCommand)&quot; -NonInteractive -NoLogo -NoProfile -Command &quot;. $(LibraryToolsFolder)\CheckChangeLog.ps1 -FilesChanged '@(FilesChanged)' &quot;"
-      ContinueOnError="false"
-      Condition=" '$(Scope)' == 'All' or '$(Scope)' == 'Latest' "/>
-    <OnError ExecuteTargets="ChangeLogErrorMessage"/>
+    <Exec Command="$(PowerShellCoreCommandPrefix) &quot;. $(LibraryToolsFolder)\CheckChangeLog.ps1 -FilesChanged '@(FilesChanged)'&quot;" ContinueOnError="false" Condition=" '$(Scope)' == 'All' or '$(Scope)' == 'Latest' " />
+    <OnError ExecuteTargets="ChangeLogErrorMessage" />
   </Target>
 
   <Target Name="ChangeLogErrorMessage">
-    <Error Text="Modified files were found with no update to their change log. Please add a snippet to the affected modules' change log."/>
+    <Error Text="Modified files were found with no update to their change log. Please add a snippet to the affected modules' change log." />
   </Target>
 
   <!-- Publish all packages -->
   <Target Name="Publish">
-    <Message Importance="high" Text="Publishing Cmdlets using $(Scope) scope"/>
+    <Message Importance="high" Text="Publishing Cmdlets using $(Scope) scope" />
 
     <Error Condition=" '$(NuGetKey)' == '' " Text="You must provide the NuGetKey parameter to the build: /p:NuGetKey=YOUR_PUBLISHING_KEY" />
 
     <Exec Command="$(PowerShellCommandPrefix) &quot;. $(LibraryToolsFolder)\PublishModules.ps1 -IsNetCore:$$(NetCore) -BuildConfig $(Configuration) -Scope $(Scope) -ApiKey $(NuGetKey) -RepositoryLocation \&quot;$(NuGetPublishingSource)\&quot; &quot; -NugetExe $(NuGetCommand)"
           Condition="'$(Scope)' != 'Stack'"
-          ContinueOnError="false"/>
+          ContinueOnError="false" />
 
     <Exec Command="$(PowerShellCommandPrefix) &quot;. $(LibraryToolsFolder)\PublishModules.ps1 -IsNetCore:$$(NetCore) -BuildConfig $(Configuration) -Scope Stack -ApiKey $(NuGetKey) -RepositoryLocation \&quot;$(NuGetPublishingSource)\&quot; &quot; -NugetExe $(NuGetCommand)"
           Condition="'$(Scope)' == 'All' or '$(Scope)' == 'Stack'"
-          ContinueOnError="false"/>
+          ContinueOnError="false" />
 
     <CallTarget Targets="BuildInstaller" Condition="'$(Scope)' =='All'" />
   </Target>
 
-  <PropertyGroup>
-    <RunTestLive Condition="'$(RunTestLive)' == ''">false</RunTestLive>
-    <XUnitIncludedTrait Condition="!$(RunTestLive)">AcceptanceType=CheckIn</XUnitIncludedTrait>
-  </PropertyGroup>
-
-  <!-- Note: all testing related target should go to 'AzurePowershell.test.targets' file except the one used by CI run -->
-  <Import Project="$(MSBuildThisFileDirectory)AzurePowershell.Test.targets"/>
-
-  <!-- Run checkin tests for each pull request -->
-  <Target Name="Test" DependsOnTargets="BeforeRunTests">
+  <Target Name="Test">
```

Comment:
Let's put back the dependency on the Build target

Reply:
`BeforeRunTests` no longer exists. Also, the current `Test` target does not depend on `Build`. It is a separate process because `Build` runs `dotnet publish` and `Test` runs a `dotnet test` (which invokes a `dotnet build` unless suppressed).